src/app/data-planes/data/index.ts

```diff
@@ -13,56 +17,138 @@ export type DataplaneWarning = {
   }
 }
 
-export function getLastUpdateTime(dataplaneOverview: DataplaneOverview): string | undefined {
-  const subscriptions = dataplaneOverview.dataplaneInsight?.subscriptions ?? []
-  if (subscriptions.length === 0) {
-    return undefined
+export type DataplaneNetworking = PartialDataplaneNetworking
+
+export type Dataplane = PartialDataplane & {
+  networking: DataplaneNetworking
+}
+
+export type DataplaneInsight = PartialDataplaneInsight & {
+  subscriptions: DiscoverySubscription[]
+  connectedSubscription?: DiscoverySubscription
+}
+
+export type DataplaneOverview = PartialDataplaneOverview & {
+  dataplane: {
+    networking: DataplaneNetworking
```

Comment:
No, it’s not a Dataplane object. It’s a `dataplane` property holding an object with the sole property `networking`. There is no point in trying to make this into a full `Dataplane` object either. The fields `mesh`, `type` (i.e. `'Dataplane'`), `name`, `creationTime`, and `modificationTime` are not useful and the case of the times not relevant because we can’t actually know them for the underlying Dataplane object.

Reply:
Oh I see this is like the 'child' one 🤔 I think these should still be explicitly named types (I think not totally sure), happy to leave this as is for now. Feel free to resolve.

Reply:
Just to clarify: what do you want to be a named typed? The `{ networking: DataplaneNetworking }` structure?

Reply:
Well to give you an example of what I was thinking over here we have an `InternalZoneIngress` type that isn't exported and is only used for the 'child'. But we use a separate type definition there.

https://github.com/kumahq/kuma-gui/pull/1867/files#diff-9fb9e8d988eecfed0988d235c43f9055bc6a50b7e5b79890f25915568a72183eR13-R16

Reply:
> what do you want to be a named typed? The `{ networking: DataplaneNetworking }` structure?

So is that a "Yes" to that second question?

(Slightly related: I removed the export and tests for the DataplaneNetworking and DataplaneInsight transformer objects. The tests are redundant and the objects aren’t meant to be used outside of this file.)

Reply:
I think its a yes, yes 😅 

We can decide/sort this after though